backend/substrapp/migrations/0001_initial.py

```diff
@@ -24,6 +24,15 @@ class Migration(migrations.Migration):
                 ('validated', models.BooleanField(default=False)),
             ],
         ),
+        migrations.CreateModel(
+            name='CompositeAlgo',
+            fields=[
+                ('pkhash', models.CharField(blank=True, max_length=64, primary_key=True, serialize=False)),
+                ('file', models.FileField(max_length=500, upload_to=substrapp.models.compositealgo.upload_to)),
+                ('description', models.FileField(max_length=500, upload_to=substrapp.models.compositealgo.upload_to)),
+                ('validated', models.BooleanField(default=False)),
+            ],
+        ),
```

Comment:
careful.
Migrations should never be modified, instead create a new one with `manage.py makemigrations`

Reply:
You are right, I planned to do that before committing, but if forgot it. Thank's.